tests/runner/src/test/resources/plugin-resources-resolver/mule-extension/META-INF/extension-manifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extension-manifest>
```

Comment:
Can you generate this file at runtime? Can be done in a separated PR if it is too much work